local/ingest.py

```diff
@@ -6,11 +6,8 @@
 
 # Data ingestion
 from backend.modules.metadata_store.client import METADATA_STORE_CLIENT
-from backend.modules.vector_db.client import VECTOR_STORE_CLIENT
-from backend.types import IngestDataToCollectionDto, DataIngestionMode
-from backend.server.services.collection import CollectionService
-from backend.modules.embedder.embedder import get_embedder
-
+from backend.types import IngestDataToCollectionDto
+from backend.server.routers.collection import ingest_data
```

Comment:
we need to refactor this. Lets shift the function to indexer folder and let backend router import it instead

Reply:
check indexer.main.py

Reply:
fixed.